ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/FSharpQuickFixBase.fs

```diff
@@ -24,10 +24,15 @@ type FSharpScopedQuickFixBase() =
 
     abstract ExecutePsiTransaction: ISolution -> unit
     default x.ExecutePsiTransaction _ = ()
+    abstract EnableFormatter: bool
```

Comment:
Could we not add complications and just do it the same way as in other quick fixes? We should either change all quick fixes (I don't see the point in the long term) or not overcomplicate this quick fix only.